pkg/reconciler/reconciler.go

```diff
@@ -953,7 +966,6 @@ func ensureDeployedRelease(u *updater.Updater, rel *release.Release) {
 	if rel.Info != nil && len(rel.Info.Notes) > 0 {
 		message = rel.Info.Notes
 	}
-	u.Update(updater.EnsureFinalizer(uninstallFinalizer))
```

Comment:
Do we use `updater.EnsureFinalizer` anywhere else? If not, it probably makes sense to remove it.

Reply:
Yeah it was used in the tests, I've changed it to somme dummy updater ^^